cmd/notary/delegations.go

```diff
@@ -249,6 +220,48 @@ func (d *delegationCommander) delegationRemove(cmd *cobra.Command, args []string
 		}
 	}
 
+	delegationRemoveOutput(cmd, d, gun, role, keyIDs)
+
+	return maybeAutoPublish(cmd, d.autoPublish, gun, config, d.retriever)
+}
+
+func delegationAddInput(d *delegationCommander, cmd *cobra.Command, args []string) (
+	config *viper.Viper, gun string, role string, keyIDs []string, error error) {
```

Comment:
also here, I'm wondering if we want named return variables since we don't seem to be using them in any particularly special way -- though, I do like how the `role` and `gun` variables are differentiated even though they're the same type.